### ISSUE TITLE
DatePicker input width

### ISSUE THREAD

**Issue by shankjn** [airbnb/react-dates#441]
Similar to daySize, can we have something like inputSize, in order to determine the width of the input for datePicker. This would help as I'm unable to import _datepicker.css locally resulting to which have to define a global width for the input.

**Comment by majapw** [comment]
Sure! I mean, it would help us in our path towards a more css in js solution... but also, the recommended path is in CSS and you should be able to do it in CSS right now.

Can you clarify the following statement?
> This would help as I'm unable to import _datepicker.css locally resulting to which have to define a global width for the input.

**Comment by shankjn** [comment]
I mean I'm able to make changes at a global level, but unable to make adjustments at component level based on the page requirements. I have webpack with css and scss loader, but still unable to import _datepicker.css. Instead have been calling at index.html. This change would make it easier to change the width at component level just like daySize.

**Comment by sjain-mi9** [comment]
It would be great if we can have something like theme as props, so that we can locally define in our components and change the styling accordingly.

**Comment by majapw** [comment]
yessss that's what I want eventually as well!

**Comment by Benno007** [comment]
Yeah the ability to pass in various variables as props without Sass would be awesome.

Our use case is we have a css loader with postcss, not using sass loader now except for this plugin.

We need to apply our theme colors for validation success/failure/untouched field and also input width, border color etc.

**Comment by majapw** [comment]
This is something I'll be working on in the next few months I think.

**Comment by jessicabyrne** [comment]
I'm wondering a similar thing -- was hoping to integrate the Datepicker component with our own input fields, since they are standard across our application. But looking at the code for SingleDatePicker it looks like the input field is included in the component. Is there an easy way to summon the calendar onClick of a custom input field?

**Comment by myfashionhub** [comment]
The input width should probably inline style. For example `Mar 30` is wider than `Apr 1` and it's a struggle to make a fixed width works seamlessly for both.
